Userland/Libraries/LibGL/GLContext.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021, Jesse Buhagiar <jooster669@gmail.com>
+ * Copyright (c) 2021, Stephan Unverwerth <s.unverwerth@gmx.de>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "SoftwareGLContext.h"
+#include <LibGfx/Bitmap.h>
+
+__attribute__((visibility("hidden"))) GLContext* g_gl_context;
+
+GLContext::~GLContext()
```

Comment:
This goes for most files: we should probably put everything that is not an official OpenGL api into a `GL` namespace like the other system libraries do.